cfe_internal/core/watchdog/templates/watchdog.mustache

```diff
@@ -141,7 +141,16 @@ if [ "${_COUNT_CF_EXECD_PROCS}" -lt "1" ]; then
 fi
 
 # Pathology #1.5: More than one cf-execd is running.
-if [ "${_COUNT_CF_EXECD_PROCS}" -gt "1" ]; then
+cf-promises --show-vars=default:sys.cf > "${COLLECTION_DIR}/cf-promises_--show-vars=default:sys.cf.txt"
+CF_VERSION_MINOR_RUNNING="$(awk '/cf_version_minor / {print $2}' /tmp/version.txt)"
+CF_VERSION_PATCH_RUNNING="$(awk '/cf_version_patch / {print $2}' /tmp/version.txt)"
```

Comment:
I guess this should read from the file produced by `cf-promises`.?

Reply:
doh

Reply:
fixed. thanks.